main.tex

```diff
@@ -50,99 +61,204 @@
 
-Bitcoin transactions transfer funds by consuming unspent outputs of previous transactions as inputs to create new outputs. The protocol rules enforced by the network ensure that transactions do not arbitrarily inflate the money supply and that outputs are spent at most once. While some newer cryptocurrencies use more sophisticated approaches to define such rules, in Bitcoin the amounts as well as the specific outputs being spent are broadcast in the clear as part of the transaction. This presents significant challenges to transacting privately\footnote{In this work we restrict the discussion of Bitcoin privacy to that of public ledger transactions, but there are other considerations especially at the network layer. For a more comprehensive discussion see \url{https://en.bitcoin.it/wiki/Privacy}.} as shown already in some of the earliest academic studies of Bitcoin~\cite{reid2013analysis,ron2013quantitative,androulaki2013evaluating,ober2013structure,moeser2013inquiry,meiklejohn2013fistful}.
-
-A fundamental requirement for electronic money is double spending prevention, and Bitcoin's main innovation was preventing double spending and illegal inflation without relying on a trusted authority and disintermediating transactions. This is in contrast to online e-cash schemes where a server authorizes transactions, or offline schemes where the identity of the double spender is revealed allowing the authority to intervene after the fact. Bitcoin's relative success suggests that the lack of trusted third parties factored more strongly in its adoption than the comparatively stronger privacy guarantees provided by the (possibly revocable) transaction  anonymity traditionally associated with e-cash~\cite{davida1997anonymity}.
+A fundamental requirement for electronic money is double-spending prevention, and Bitcoin's main innovation was preventing double-spending and illegal inflation without relying on a trusted authority, thereby disintermediating transactions. This is in contrast to online e-cash schemes where a server authorizes transactions or offline schemes where the identity of the double spender is revealed allowing the authority to intervene after the fact. Bitcoin's relative success in comparison suggests that the lack of trusted third parties factored more strongly in its adoption than the comparatively stronger privacy guarantees provided by the (possibly revocable) transaction  anonymity traditionally associated with e-cash~\cite{davida1997anonymity}.
```

Comment:
`Digital Bearer Certificate [DBC]` is afaik the neutral term for these things. `e-Cash` was an early implementation of a DBC by Chaum.

```suggestion
A fundamental requirement for electronic money is double-spending prevention, and Bitcoin's main innovation was preventing double-spending and illegal inflation without relying on a trusted authority, thereby disintermediating transactions. This is in contrast to digital bearer certificates schemes where a server authorizes transactions or offline schemes where the identity of the double spender is revealed allowing the authority to intervene after the fact. Bitcoin's relative success in comparison suggests that the lack of trusted third parties factored more strongly in its adoption than the comparatively stronger privacy guarantees provided by the (possibly revocable) transaction  anonymity traditionally associated with e-cash~\cite{davida1997anonymity}.
```

Reply:
in the literature being cited here "online e-cash" and "offline e-cash" are standard terms, i think DBC only came about after the blockchain fad started